npmpackagejsonlint.config.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+	// See https://www.npmjs.com/package/@wordpress/npm-package-json-lint-config
+	extends: '@wordpress/npm-package-json-lint-config',
+	rules: {
+		'prefer-no-devDependencies': 'error',
+		'prefer-property-order': 'off',
+		'require-bugs': 'off',
+		'require-homepage': 'off',
+		'require-keywords': 'off',
+		'valid-values-author': [ 'error', [ 'Automattic Inc.' ] ],
+		'valid-values-name-scope': [ 'error', [ '@automattic' ] ],
+	},
+	overrides: [
+		{
+			patterns: [
+				'./apps/full-site-editing/package.json',
```

Comment:
any reason these are explicit instead of `./apps/*/package.json`?

Reply:
No other than each app is a bit different and not all of them have package-lock currently. Happy to have it more generic. 👍

Reply:
do they need a lock to be linted?

Reply:
fine for now